resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -52,6 +52,10 @@ jruby-puppet: {
     # specified in the Puppet Server HOCON-formatted auth.conf (if false or not
     # specified).
     #use-legacy-auth-conf: true
+
+    # (optional) Whether or not to track hiera lookups during compilation; turning
+    # this on will send that information to puppetdb
+    # track-hiera-lookups: true
```

Comment:
piggybacking on Patrick's previous comment, should we mention hiera here? (I know the function moved from `heira()` to `lookup()`. I would assume `track-lookups` would be sufficient?

Reply:
Commit has been added to change to remove hiera language.